cli/js/dispatch_minimal_test.ts

```diff
@@ -10,7 +10,7 @@ const readErrorStackPattern = new RegExp(
   `^.*
     at unwrapResponse \\(.*dispatch_minimal\\.ts:.*\\)
     at Object.sendAsyncMinimal \\(.*dispatch_minimal\\.ts:.*\\)
-    at async Object\\.read \\(.*files\\.ts:.*\\).*$`,
+    at async Object\\.read \\(.*io\\.ts:.*\\).*$`,
   "ms"
 );
```

Comment:
Not actionable for this PR, just a comment: This sort of call stack inspection, especially with an RegExp does not belong here. AFAIK async stack trace is broken anyway - so this test probably does nothing.

Reply:
I'm not sure, this stack trace seems valid to me, I have a feeling that async stack traces are only broken in some conditions. Anyway, this needs to be sorted out 👍